packages/docusaurus/bin/docusaurus.mjs

```diff
@@ -137,6 +137,7 @@ cli
   .option(
     '--poll [interval]',
     'use polling rather than watching for reload (default: false). Can specify a poll interval in milliseconds',
+    parseInt,
```

Comment:
```suggestion
```

we also support `--poll` atm, and the option can be just "true" without a value

Reply:
Without _some_ kind of input transformation it will always be a string, so it can't be directly done that way.

Reply:
that's not what I see in practice here

<img width="563" alt="CleanShot 2022-04-20 at 14 35 31@2x" src="https://user-images.githubusercontent.com/749374/164231573-c883fff7-da0e-4bbd-9de3-b66cc546ac0e.png">